cypress/support/e2e.ts

```diff
@@ -1 +1,15 @@
-import "@testing-library/cypress/add-commands";
\ No newline at end of file
+import "@testing-library/cypress/add-commands";
+
+// This is a workaround for a bug in Cypress: https://github.com/remix-run/remix/issues/2570#issuecomment-1259089527
```

Comment:
@eladroz Unfortunately, it still need it. By removing it, I got the following error: 

```
Error: The following error originated from your application code, not from Cypress.

  > Hydration failed because the initial UI does not match what was rendered on the server.
```

Even in the official template from the Remix team have it: https://github.com/remix-run/indie-stack/blob/main/cypress/support/e2e.ts